crates/astria-conductor/src/conductor.rs

```diff
@@ -36,32 +37,41 @@ use crate::{
     Config,
 };
 
+/// Token to signal whether the conductor should restart or shut down.
+pub enum RestartToken {
```

Comment:
It should not be public, and we should probably reserve the use of the word `Token` for cancellation tokens:

```suggestion
enum RestartOrShutdown {
```